canonical-web-and-design/ubuntu.com
# Copy Update on azure/pro

## Done

- Update What's in Ubuntu Pro matrix as per copy doc 

## QA

- View page at: https://ubuntu-com-9212.demos.haus/azure/pro
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/1WaQ5n3GxwZzbX0T6rLiRNejFvxIPIHd5V5XgdAwKmSw/edit) and [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/9204) 


## Issue / Card

Fixes #9204 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/107210742-77f2ad80-69fc-11eb-89fb-9a270d554b44.png)
